op.c

```diff
@@ -7899,20 +7899,18 @@ Perl_utilize(pTHX_ int aver, I32 floor, OP *version, OP *idop, OP *arg)
 
         U16 shortver = S_extract_shortver(aTHX_ use_version);
 
-        /* If a version >= 5.11.0 is requested, strictures are on by default! */
-        if (shortver >= SHORTVER(5, 11)) {
+        if (shortver >= SHORTVER(5, 37)) {
+            PL_hints |= HINT_STRICT_REFS | HINT_EXPLICIT_STRICT_REFS |
+                HINT_STRICT_SUBS | HINT_EXPLICIT_STRICT_SUBS |
+                HINT_STRICT_VARS | HINT_EXPLICIT_STRICT_VARS;
+        } else if (shortver >= SHORTVER(5, 11)) {
```

Comment:
The style guide in perlguts says to use uncuddled elses (also below)

Reply:
Ohyes, I just copied that from Zefram's patch. Shall fix.